Resources/Prototypes/Nuclear14/Entities/Structures/Wallmount/adverts.yml

```diff
@@ -53,6 +101,50 @@
   - type: Sprite
     state: nukagirl-worn
     scale: 0.5, 0.5
+
+- type: entity
+  parent: N14SignBase
+  id: N14SignVaultTecWorn
+  suffix: worn
+  name: Vault-Tec advert
+  description: "A pre-war advert for Vault-Tec."
+  components:
+  - type: Sprite
+    state: vaulttec-worn
+    scale: 0.5, 0.5
+    
+- type: entity
+  parent: N14SignBase
+  id: N14SignVaultTecWorn2
+  suffix: damaged
+  name: Vault-Tec advert 2
```

Comment:
Name is player facing so it can be a duplicate (ID is the only thing that you can’t duplicate). IIRC name and description are inherited so you could parent this one to the other vault tec advert.